src/index.js

```diff
@@ -40,21 +38,45 @@ let IPFS
 let mb
 let shouldPoll = false
 const statsCache = {}
+const locationsCache = {}
 
 function pollStats (ipfs) {
-  const next = () => {
-    setTimeout(() => {
-      pollStats(ipfs)
-    }, 1000)
-  }
+  const next = () => setTimeout(() => pollStats(ipfs), 1000)
 
   if (!shouldPoll || !mb.window || !mb.window.isVisible()) {
     return next()
   }
 
   ipfs.swarm.peers()
     .then((res) => {
-      statsCache.peers = res.length
+      res = res.sort((a, b) => {
+        return a.peer.toB58String() > b.peer.toB58String() ? 1 : -1
```

Comment:
You can just do `return a.peer.toB58String() > b.peer.toB58String()`, it will already return true/false

Reply:
IDK why I did that 😆